tests/acceptance/26_cf-net/serial/cf-net_connect.cf

```diff
@@ -0,0 +1,52 @@
+body contain myshell
+{
+    !windows::
+        useshell => "useshell";
+    windows::
+        useshell => "powershell";
+}
+
+body common control
+{
+    inputs => {
+        "../../default.cf.sub",
+        "../../run_with_server.cf.sub"
+    };
+    bundlesequence => { default("$(this.promise_filename)") };
+}
+
+bundle agent init
+{
+    methods:
+        "any" usebundle => dcs_fini("$(G.testdir)/cf-net_connect.txt");
+
+        "any" usebundle => generate_key;
+        "any" usebundle => trust_key;
+
+        "any" usebundle => start_server("$(this.promise_dirname)/cf-net_connect.cf.srv");
+}
+
+bundle agent test
+{
+    commands:
+        "$(sys.cf_net)"
+            args => " -H 127.0.0.1:15000 connect > $(G.testdir)/cf-net_connect.txt",
```

Comment:
You probably want to catch standard error as well, to make sure no unwanted errors are spewed in the output. I would also rename the file to indicate it's storing output, `.txt` too generic. Something like this:

```
args => " -H 127.0.0.1:15000 connect   > $(G.testdir)/cf-net_connect.log  2>&1", 
```